python/examples/minimal_build/build_conda.sh

```diff
@@ -97,9 +97,6 @@ export CMAKE_PREFIX_PATH=${ARROW_HOME}${CMAKE_PREFIX_PATH:+:${CMAKE_PREFIX_PATH}
 export PYARROW_BUILD_TYPE=Debug
 export PYARROW_CMAKE_GENERATOR=Ninja
 
-# You can run either "develop" or "build_ext --inplace". Your pick
-
-# python setup.py build_ext --inplace
-python setup.py develop
+python setup.py build_ext --inplace
```

Comment:
We'd really like this to become `pip install -e`, if possible. AFAIU, invoking `setup.py` directly is heavily discouraged nowadays.

Reply:
Oh, I see you changed this while I was commenting :-) Nevermind!

Reply:
I just pushed a change with:
```
# Use the same command that we use on python_build.sh
python -m pip install --no-deps --no-build-isolation -vv .
```